src/Sarif/Baseline/DefaultBaseline/PhysicalLocationBaselineEquals.cs

```diff
@@ -17,8 +17,8 @@ public bool Equals(PhysicalLocation x, PhysicalLocation y)
                     return false;
                 }
 
-                // Only compare URI (so file path/relative file path).  UriBaseId and Region may change run over run.
-                if (x.Uri != y.Uri)
+                // Only compare URI (so file path/relative file path). UriBaseId and Region may change run over run.
```

Comment:
No. As in a previous review: `UriBaseId` is a token like `SRCROOT`. The absolute URI the token refers to might change over time, but that token won't.

Why would `Region` change?

Reply:
Sorry I thought I had done this correctly this time...